static_src/test/functional/pageobjects/user_invite.element.js

```diff
@@ -10,14 +10,17 @@ import BaseElement from './base.element';
 
 export default class UserInviteElement extends BaseElement {
   inputToInviteForm(input) {
+    browser.waitForExist('.test-users_invite_name');
     return this.element('.test-users_invite_name').setValue(input);
   }
 
   getInviteFormValue() {
+    browser.waitForExist('.test-users_invite_name');
     return this.element('.test-users_invite_name').getValue();
   }
 
   submitInviteForm() {
+    browser.waitForExist('[type="submit"]');
```

Comment:
in this case, you're searching the whole DOM for `[type="submit"]`. Below, it uses `this.element` so it's only at the element. We technically have to scope it to just the element, maybe like: `.test-user_invite [type="submit"]`.